specs/src/specs/header-sync.md

```diff
@@ -0,0 +1,37 @@
+# P2P Header Sync
+
+## Abstract
+
+The P2P Header Sync is a p2pP exchange service for rollkit headers that implements the go-header interface. The main components are:
```

Comment:
i'd link to the go header repo. We can use re-useable markdown links to keep the specs easier to read, like so

```suggestion
The P2P Header Sync is a p2pP exchange service for rollkit headers that implements the [go-header][go-header] interface. The main components are:
```

at the bottom of the file
```
[go-header]: https://github.com/celestiaorg/go-header
```